types/coingecko-api/index.d.ts

```diff
@@ -236,6 +240,12 @@ interface CoinsFetchHistoryData {
     public_interest_stats: { alexa_rank: number; bing_matches: null };
 }
 
+interface CoinsFetchMarketChart {
+    market_caps: number[][];
```

Comment:
We can actually go a step further for all of these as we know we're comparing a coin against a different currency so we get pairs, e.g.:

```suggestion
    market_caps: [number, number][];
```

Reply:
it's a great idea but I'm not sold yet whether I prefer "tuples" over arrays. Since it's not critical, I'll keep it as is and will consider updating it later on.